build.xml

```diff
@@ -6,10 +6,63 @@
         http://www.erpya.com
     </description>
   <!-- set global properties for this build -->
+  <property name="env_directory" location="adempiere/Adempiere/utils"/>
+  <property name="AD_directory" location="adempiere/Adempiere"/>
   <property name="adempiere_build" location="utils_dev/build.xml"/>
+  <property name="myEnvironment_CI" location="adempiere/Adempiere/utils/myEnvironment_CI.sh"/>
+  <property name="myEnvironment" location="adempiere/Adempiere/utils/myEnvironment.sh"/>
+  <property name="RUN_DBRestore" location="adempiere/Adempiere/utils/RUN_DBRestore.sh"/>
+  <property name="RUN_DBExport" location="adempiere/Adempiere/utils/RUN_DBExport.sh"/>
+  <property name="RUN_ImportAdempiere" location="adempiere/Adempiere/utils/RUN_ImportAdempiere.sh"/>
+  <property name="RUN_MigrateXML" location="adempiere/Adempiere/utils/RUN_MigrateXML.sh"/>
+  <property name="RUN_silentsetup" location="adempiere/Adempiere/RUN_silentsetup.sh"/>
+  <property name="AdempierePropertiesFrom" location="adempiere/Adempiere/utils/adempiereProperties_CI.properties"/>
+  <property name="AdempierePropertiesTo" location="adempiere/Adempiere/Adempiere.properties"/>
+  <property name="myDBcopyTemplate" location="adempiere/Adempiere/utils/myDBcopyTemplate.sh"/>
+  <property name="myDBcopy" location="adempiere/Adempiere/utils/myDBcopy.sh"/>
+  
```

Comment:
This is great!  For the future (a follow on PR), it would be good to add RUN_UnitTests, RUN_IntegrationTests and RUN_AllTests.

Reply:
I see and all test are inside util_dev/build.xml and the main build called from action just run the util_dev/biuild.xml with complete target and all test are running.

Fiiine